### ISSUE TITLE
Upload -  lack of response [beta]

### ISSUE THREAD

**Issue by sr320** [uwescience/sqlshare#46]
In attempting to upload a very large file (8GB) I can select the file but nothing happens. 
(Chrome and Safari)

I imagine this might be over some limit, however I would suggest that if this is the case you
maybe insert text below `Accepted file types` to indicate file size limit. 

![upload_file](https://cloud.githubusercontent.com/assets/4442039/6691690/3abdb740-cc85-11e4-8c5d-1fa1c48eea00.png)
# beta

**Comment by shrjain** [comment]
Your limit is 1TB, I just verified, so this has to be something else. @vegitron, any thoughts?

**Comment by vegitron** [comment]
I bet the uploader is trying to load the whole file into memory, and send it in one shot.  It looks like the options have greatly improved since we last looked at file uploading, and we should be able to do much smarter file slicing and uploading of chunks.  There's http://www.resumablejs.com/ for example.

**Comment by dhalperi** [comment]
But, for now, you should be able to use the command-line uploader for big
files. (Shrainik?)

---

Daniel Halperin
Director of Research for Scalable Data Analytics
eScience Institute
University of Washington

**Comment by shrjain** [comment]
Yes, that's correct. @sr320, can you try the upload with the pythonclient (with the instructions I sent in the original email)?

**Comment by billhowe** [comment]
I'd be supportive of adding browser uploads for big files to the upcoming
sprints!

On Tue, Mar 17, 2015 at 9:25 AM, Patrick Michaud notifications@github.com
wrote:

> I bet the uploader is trying to load the whole file into memory, and send
> it in one shot. It looks like the options have greatly improved since we
> last looked at file uploading, and we should be able to do much smarter
> file slicing and uploading of chunks. There's http://www.resumablejs.com/
> for example.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/uwescience/sqlshare/issues/46#issuecomment-82447288.

## 

Bill Howe
Associate Director and Senior Data Science Fellow, UW eScience Institute
Affiliate Faculty, Computer Science & Engineering
University of Washington
To acknowledge eScience: "Supported in part by the University of Washington
eScience Institute"

**Comment by katimcisse** [comment]
BillH - I added that feature to the Phase2 scope/backlog https://docs.google.com/a/uw.edu/spreadsheets/d/1nZnYZcDwd2VLoormlXOOTp2ThbzI6aKHIxUwH9WWqTA/edit?pli=1#gid=392793894

**Comment by chickadeedee** [comment]
This bug, and associated bugs are already in the jira backlog.

Nina Bohn, QA Lead
Academic and Collaborative Applications, UW-IT
tbohn@uw.edu

On Mar 17, 2015 11:19 AM, Bill Schaefer notifications@github.com wrote:

BillH - I added that feature to the Phase2 scope/backlog https://docs.google.com/a/uw.edu/spreadsheets/d/1nZnYZcDwd2VLoormlXOOTp2ThbzI6aKHIxUwH9WWqTA/edit?pli=1#gid=392793894

##
